app/views/registration/steps/_type.html.haml

```diff
@@ -6,9 +6,9 @@
     .row
       .small-12.columns
         %header
-          %h2{ "ng-bind" => "'enterprise.registration.modal.steps.type.headline' | t:{enterprise: enterprise.name}" }
+          %h2{ "ng-bind" => "'registration.steps.type.headline' | t:{enterprise: enterprise.name}" }
```

Comment:
is there a way I can use rails t function with lazy lookup in this line?

Reply:
Good question. I had to think about that for a moment. First I thought no, because `enterprise.name` is stored in the browser only, it doesn't exist in Rails yet. But now I think you can use Angular's `{{ enterprise.name }}` to print it:
```haml
%h2{ "ng-bind" => t(".headline", enterprise: "{{enterprise.name}}") }
```

Reply:
awesome, thanks. I dont think I need the binding then:
%h2= t(".headline", enterprise: "{{enterprise.name}}")
it works.